vendor/github.com/Microsoft/go-winio/backup.go

```diff
@@ -230,10 +219,11 @@ type BackupFileWriter struct {
 	ctx             uintptr
 }
 
-// NewBackupFileWriter returns a new BackupFileWriter from a file handle. If includeSecurity is true,
+// NewBackupFileWrtier returns a new BackupFileWriter from a file handle. If includeSecurity is true,
```

Comment:
Writer

Reply:
Ah it's library code. Ok. Never mind :)

Reply:
It's vendor code. We should not modify it.